src/topology/builder.rs

```diff
@@ -112,14 +112,13 @@ pub async fn load_enrichment_tables<'a>(
 }
 
 pub struct Pieces {
```

Comment:
I get the sense that there's a `pub` function or something elsewhere in the source tree that is keeping `Pieces` public.